packages/terra-functional-testing/jest.config.js

```diff
@@ -5,8 +5,4 @@ module.exports = {
   watchPathIgnorePatterns: [
     './tests/jest/reports/results/terra-functional-testing.json',
   ],
-  testEnvironment: '@cerner/jest-config-terra/lib/JestEnvironmentJsdomTerra.js',
-  transform: {
-    '^.+\\.(js|jsx)$': 'babel-jest',
-  },
```

Comment:
This was removed as they are basically the same as the default config from `jest-config-terra`.